test/cljs/status_im/test/i18n.cljs

```diff
@@ -1,6 +1,29 @@
 (ns status-im.test.i18n
   (:require [cljs.test :refer-macros [deftest is]]
-            [status-im.i18n :as i18n]))
+            [status-im.i18n :as i18n]
+            [clojure.set :as set]
+            [cljs.spec.alpha :as spec]))
 
 (deftest label-options
   (is (not (nil? (:key (i18n/label-options {:key nil}))))))
+
+(deftest locales-only-have-existing-tran-ids
+  (is (spec/valid? ::i18n/trans-ids (i18n/trans-ids-for-all-locales))
+      (->> i18n/locales
```

Comment:
I think we need supported-locales here as well otherwise the tests won't pass and the app won't build

Reply:
This test is failing because some locales have non-existent labels (based on English's labels). I think that once the PR is approved we need to remove those dead labels to achieve consistency in labels and make this test pass. Let me know if you want to proceed some other way though.